app/scripts/controllers/eventDetails.js

```diff
@@ -81,6 +81,9 @@ angular
     $scope.originalConference = conference;
     $scope.conference = angular.copy(conference);
     $scope.currencies = currencies;
+    $scope.conference.locationCountry = conference.locationCountry
+      ? conference.locationCountry
+      : 'US';
```

Comment:
I think the API will set this to ```''```(@Benjamin-deVries can confirm) by default, so it's up to the UI to set the value to 'US'.

Reply:
Indeed, the API's default behavior will be an empty string i.e. when the JSON is marshalled into a ConverenceEntity object on the API side, if there are fields that are omitted they will default to an empty string.